custom/auth/ante/ante.go

```diff
@@ -13,15 +13,16 @@ import (
 
 // HandlerOptions are the options required for constructing a default SDK AnteHandler.
 type HandlerOptions struct {
-	AccountKeeper      cosmosante.AccountKeeper
-	BankKeeper         BankKeeper
-	FeegrantKeeper     cosmosante.FeegrantKeeper
-	OracleKeeper       OracleKeeper
-	TreasuryKeeper     TreasuryKeeper
-	SignModeHandler    signing.SignModeHandler
-	SigGasConsumer     cosmosante.SignatureVerificationGasConsumer
-	IBCChannelKeeper   channelkeeper.Keeper
+	AccountKeeper    cosmosante.AccountKeeper
```

Comment:
go fumpt ftw :D We should probably find some way to make a commit hook to automatically run go fumpt when we commit to our local repos

Reply:
https://github.com/edsrzf/gofmt-git-hook might be something we can adapt to work for our needs

Reply:
🤣 yeah. That's probably a good idea. I never bother too much aboit gofumpting until I completed the work. I still have to push these three tests of the ante handler.... After that I will clean.

Reply:
linter is happy